devdocs/orientation-guide.md

```diff
@@ -0,0 +1,131 @@
+<center>![](images/teammates_heading.png) </center>
+
+
+# **Contributor Orientation Guide**
+
+
+
+We welcome contributions from anyone, in particular, students (see [here](https://www.google.com/url?q=http://teammatesv4.appspot.com/about.html&sa=D&ust=1458427634357000&usg=AFQjCNGIbS5nfk0HxVCMSqPXXmcaOSz47w) for a list of our contributors). One of the main objectives of TEAMMATES is to help students get experience in a OSS production environment. Here are some information that might be useful to would-be contributors.
+
+
+
+[Knowledge required](#h.abftcg92xk1k)
```

Comment:
These four links are not correct. Please update them so that they will link to the appropriate sections